tendrl/commons/objects/atoms/__init__.py

```diff
@@ -19,11 +19,11 @@ def __init__(
                                              self.obj.__name__)
         atom_def = obj_def.atoms[self.__class__.__name__]
 
-        self.inputs = inputs or atom_def['inputs']['mandatory']
-        self.enabled = enabled or atom_def['enabled']
-        self.outputs = outputs or atom_def['outputs']
-        self.uuid = uuid or atom_def['uuid']
-        self.help = help or atom_def['help']
+        self.inputs = inputs or atom_def.get('inputs')['mandatory']
```

Comment:
atom_def.get('inputs')['mandatory'] should be atom_def.get('inputs').get("mandatory")